Implementation/Services/index.md

```diff
@@ -232,10 +240,11 @@ namespace Umbraco8.Extensions
 {
     public static class UmbracoHelperExtensions
     {
-            //NewsCategories //NewsSection
-            public static IPublishedContent GetNewsSection(this UmbracoHelper umbracoHelper)
+        public static IPublishedContent GetNewsSection(this UmbracoHelper umbracoHelper)
             {
-            return umbracoHelper.ContentSingleAtXPath("root/homePage/newsSection");
+                //assuming a single site with a single News Section at the top level
+                IPublishedContent siteRoot = umbracoHelper.ContentAtRoot().FirstOrDefault();
+                return siteRoot != null ? siteRoot.FirstChild(f => f.ContentType.Alias == "newsSection") : default(IPublishedContent);
```

Comment:
I don't think we should use `default(IPublishedContent)`, it should just be `null`. `IPublishedContent` isn't a struct so this equals null anyways.

Reply:
This can be rewritten like, but maybe you are keeping the syntax explicit for a reason?

```cs
var newsSection = siteRoot.?siteRoot.FirstChild(f => f.ContentType.Alias == "newsSection") ?? null;
```

Reply:
updated, you 'cool kids' with your fancy c# 6 syntax... it will never catch on :-P have added a comment above to be clear, and updated other instances of same.